components/templates/NewsPage/index.tsx

```diff
@@ -0,0 +1,62 @@
+import { FunctionComponent } from "react";
+import sanitizeHtml from "sanitize-html";
+import { ReleasesService } from "@/lib/api/noirlab/codegen";
+import { Locale } from "@/lib/i18n/settings";
+import NewsPageClient from "./client";
+
+const NewsPage: FunctionComponent<{
+  section: string;
+  data: PageEntry;
+  locale: string;
+}> = async ({ data, section, locale }) => {
+  const { pressReleaseId } = data;
+
+  if (pressReleaseId) {
```

Comment:
this server component fetches press release data from the Noirlab API and then mixes it into the news article props that are passed to the client component